Documentation/Metrics/arangodb_vocbase_meta_collection_lock_locked_exclusive.yaml

```diff
@@ -0,0 +1,14 @@
+name: arangodb_vocbase_meta_collection_lock_locked_exclusive
+introducedIn: "3.12.1"
+help: |
+  Number of exclusively locked collections.
+unit: number
+type: gauge
+category: Transactions
+complexity: advanced
+exposedBy:
+  - dbserver
+  - agent
+  - single
+description: |
+  Counts the exclusively locked collection across all shards in a database.
```

Comment:
```suggestion
  Counts the exclusively locked collections across all shards in a database.
```

Reply:
Maybe also note that on a DB server, the number reported by the metric is the number of exclusively locked _shards_, not _collections_.